docker-compose.override.yaml

```diff
@@ -27,9 +34,20 @@ services:
       - 8081
     ports:
       - "127.0.0.1:8081:8081"
+    environment:
+      - DJANGO_SECRET_KEY=secretkey
+      - DEFAULT_FILE_STORAGE=django.core.files.storage.FileSystemStorage
+      - GOOGLE_APPLICATION_CREDENTIALS=/var/akvo/rsr/code/akvo/akvo-lumen-service-account-credentials.json
+    healthcheck:
+      test: [ "CMD", "./manage.py", "probe_django_q" ]
+      interval: 1m30s
+      timeout: 10s
+      retries: 3
+      start_period: 30s
     volumes:
       - .:/var/akvo/rsr/code:delegated
       - /var/akvo/rsr/code/src/
+      - ./scripts/docker/dev/50-docker-local-dev.conf:/config_overrides/50-docker-local-dev.conf
```

Comment:
I don't think the `DEFAULT_FILE_STORAGE` and `GOOGLE_APPLICATION_CREDENTIALS` are necessary for the dev env

Reply:
I removed `GOOGLE_APPLICATION_CREDENTIALS`. I think `DEFAULT_FILE_STORAGE` is necessary to store uploaded files on the disk instead of attempting to contact use a google bucket